src/main/java/random_csapatnev/Main.java

```diff
@@ -18,15 +18,16 @@
  */
 public class Main implements Serializable
 {
+	private static String fontstr = "Arial";
```

Comment:
Javaban a preferált konstans elnevezés azt hiszem FONTSTR - illetve következő iterációban ez is refaktorálható lehet a többi magic value-val egy helyre.

Reply:
[c380e91](https://github.com/BME-MIT-IET/iet-hf-2023-grafadatbazis/pull/14/commits/c380e9172c125819a458b7c3964a396f8d0b03ce) commitban javítva.